src/axom/sidre/core/MFEMSidreDataCollection.cpp

```diff
@@ -2636,11 +2636,11 @@ void MFEMSidreDataCollection::reconstructField(Group* field_grp)
     }
 
     // We cache the FESpaces to avoid reconstructing them when not needed
-    // An FESpace is uniquely identified by the basis of its FEColl and its ordering
+    // An FESpace is uniquely identified by the basis of its FEColl, ordering, and vdim number
     const std::string fespace_id =
-      axom::fmt::format("{0}_{1}",
+      axom::fmt::format("{0}_{1}_{2}",
                         basis_name,
-                        (ordering == mfem::Ordering::byVDIM) ? "nodes" : "vdim");
+                        (ordering == mfem::Ordering::byVDIM) ? "vdim" : "nodes", vdim);
```

Comment:
I think you need to run `make style` in a config w/ clang@10

Reply:
As a shortcut -- this is what it wants to see:

```diff
     const std::string fespace_id =
       axom::fmt::format("{0}_{1}_{2}",
                         basis_name,
-                        (ordering == mfem::Ordering::byVDIM) ? "vdim" : "nodes", vdim);
+                        (ordering == mfem::Ordering::byVDIM) ? "vdim" : "nodes",
+                        vdim);
```
https://dev.azure.com/healy22/Status%20Check%20Testiing/_build/results?buildId=8442&view=logs&j=cb13b0c9-f406-547a-d755-9fe006f85343&t=435ca3cd-a81f-530c-87da-e79a43d1b3d6&l=511